cmd/osctl/cmd/time.go

```diff
@@ -52,9 +54,10 @@ var timeCmd = &cobra.Command{
 				helpers.Fatalf("error parsing remote time: %s", err)
 			}
 
-			fmt.Printf("NTP Server: %s\n", output.Server)
-			fmt.Printf("Local time: %s\n", localtime)
-			fmt.Printf("Remote time: %s\n", remotetime)
+			w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
+			fmt.Fprintln(w, "SERVER\tLOCAL-TIME\tREMOTE-TIME")
```

Comment:
should it be `NTP-SERVER` to make it less confusing with whatever 'server' might be?

Reply:
Yes, good point